source/technical/shares.rst

```diff
@@ -0,0 +1,234 @@
+the default quota, please submit a ticket via the `Help Desk <https://chameleoncloud.org/user/help/>`_.
+
+Managing Shares using GUI
+================================
+
+To manage your share, use the `Shares` page at `CHI@UC <https://chi.uc.chameleoncloud.org/>`_ or `CHI@TACC <https://chi.tacc.chameleoncloud.org/>`_
+by navigating to `Project > Share > Shares`.
+
+  .. figure:: shares/sharespage.png
+      :alt: The Shares page
+
+      The Shares page
+
+Create Share
+--------------------------------
+
+Click the `Create Share` button. In the `Create Share` dialog, provide a name and the size of your share, and then click the `Create` button to
+create a share.
+
+  .. figure:: shares/createshare.png
+      :alt: The Create Share dialog
+
+      The Create Share dialog
+
+  .. note::
+  
+    The storage VLAN is not required for creating shares. It’s only required to access the shares.
+
+.. _view-share-gui:
+
+View Share
+--------------------------------
+
+You can look at the details of a share by clicking the share name in the `Shares` page. Note that the paths of the `export locations` are important
+as you will use this path to mount your share to your bare metal instances. You can also see the other properties, such as visibility and size.
+The access rules are listed in the `share details` page, though you can not edit the rules, as they are controlled by the reservation service.
+
+  .. figure:: shares/sharedetails.png
+      :alt: The Share details
+
+      The Share details
+      
+Edit Share
+--------------------------------
+You can manage the properties and extend the size of a share by clicking the `Action` dropdown in the `Shares` page. 
+
+  .. figure:: shares/manageshare.png
+      :alt: The Action dropdown
+
+      The Action dropdown
+
+Delete Share
+--------------------------------
+You can use the `Action` dropdown to delete a single share, or select multiple shares and click the `Delete Shares` button. 
+
+  .. important::
+  
+    Be careful when deleting shares, as the action is irreversible. However, the termination of your VLAN reservation **DOES NOT** delete your share.
+    Your shares persist until you manually delete them.
+
+
+Managing Shares using CLI
+================================
+
+As all other Chameleon services, you can manage your shares via CLI as well. 
+
+  .. tip::
+  
+    Reading :ref:`Command Line Interface (CLI) <cli>` is highly recommended before continuing on the following sections.
+
+In addition to installing the CLI, you must also install `python-manilaclient` package:
+
+  .. code-block:: bash
+
+    pip install python-manilaclient
+
+Then, you must set environment variables for your account and project using :ref:`The OpenStack RC Script <cli-rc-script>`.
+
+List Shares
+--------------------------------
+
+To list all shares of your project, run the following command:
+
+  .. code-block:: bash
+
+    manila list
```

Comment:
In general I think we should prefer to use the `openstack` flavor of commands, is there one for manila?